libs/npm/core/src/utils/elastic/generateQueryValues.ts

```diff
@@ -65,6 +65,6 @@ export const generateQueryValues = (
           moment(settings.updatedOn).endOf('day'),
         )
       : undefined,
-  ].filter((v) => v !== undefined);
+  ].filter((v) => (v !== undefined && !Array.isArray(v)) || (Array.isArray(v) && v.length > 0));
```

Comment:
Temporary test to see if this resolves some issues Felipe is running into.  I see no reason why we would send empty arrays to elasticsearch.